parquet/src/file/metadata.rs

```diff
@@ -303,9 +309,18 @@ impl RowGroupMetaData {
             let cc = ColumnChunkMetaData::from_thrift(d.clone(), c)?;
             columns.push(cc);
         }
+        let sorting_columns: Option<Vec<SortingColumnMetaData>> = match rg.sorting_columns
+        {
+            Some(sorting_columns) => {
+                let result = sorting_columns.iter().map(Into::into).collect();
```

Comment:
```suggestion
                let result = sorting_columns.iter().copied().map(Into::into).collect();
```

To match suggested changes to From